composer.json

```diff
@@ -122,7 +122,9 @@
         "phpunit/phpunit": "4.5.0",
 
         "mmoreram/php-formatter": "1.0.4",
-        "fabpot/php-cs-fixer": "1.4.2"
+        "fabpot/php-cs-fixer": "1.4.2",
+        "visithor/visithor-bundle": "0.1.*@dev",
```

Comment:
not true, not if you lock it to the third digit and not if you don't use @dev. If you use @dev you are using dev-master here which i think is worst no?